### ISSUE TITLE
Set up a staging instance

### ISSUE THREAD

**Issue by mpreisler** [OpenSCAP/website#142]
Right now we are doing the following super nasty things:
1) installing brand new untested plugins on a live site (!!!)
2) changing the source code of wordpress trying to workaround an issue

Lets be more responsible and more careful about the site, if we want new complex functionality deployed, let's first test it in a staging area.

I propose this:
create www-staging-rhopenscap.rhcloud.com where we can test. Once the test is a success and everything works we can snapshot it and restore on the live site.

**Comment by lhorakov** [comment]
We are not changing the code now. It will be https. Plugins are these:
Added  by Lenka Horakova:
404 to 301 - for redirection - needed/don't remove
Google XML Sitemaps  - for google search - needed/don't remove
Sitemap - for real sitemap in our page - needed/don't remove

Added by Unknown(please change if you know):
Broken Link Checker
List category posts
Proxy Real IP
Redirection
Simple Analytics
Ultimate TinyMCE
WP RSS Aggregator

Please, actualize the list. Than we will know what can be possibly removed after snapshot.

**Comment by mpreisler** [comment]
`404 to 301` is actually unused, we use the `Redirection` plugin added by me.

I added all the plugins you listed except `Ultimate TinyMCE`, I have no idea who added that and AFAIK it's not needed.

I propose to make a snapshot and then remove `404 to 301` and `TinyMCE`.

**Comment by lhorakov** [comment]
It's deleted now and it seems to me that everything is working, even 404. For me to close this.

**Comment by mpreisler** [comment]
We haven't solved the issue that's described here, we have just done a minor plugins clean-up. We can't close this yet.

**Comment by lhorakov** [comment]
But your issue was, that you thought that we are changing some code in wordpress what we are not doing.
And your first point were plugins, but we are also not adding plugins for nothing and new ones.

**Comment by mpreisler** [comment]
Yeah, since we are not adding plugins or changing source code anymore, we can postpone this. If at some later point we want to do big changes we will need to have some sort of a staging instance.
